water-plants/src/actions/index.js

```diff
@@ -40,6 +40,8 @@ export const registerUser = registration => dispatch => {
 
 export const login = creds => dispatch => {
   dispatch({ type: LOGIN_START });
+  localStorage.removeItem('token');
```

Comment:
makes sense, was thinking of a similar functionality for some parts of state as well